components/DataTable/TableBody/Rows.tsx

```diff
@@ -26,25 +26,19 @@ export default function Rows({rows, page, pageSize}: RowsProps) {
   return (
     <>
       {
-        pageOfRows.map((row: any, rowIndex: number) => {
-          let rowProps: Object = {
-            className: 'rowStyle',
-            id: `row-id-${rowIndex}`,
-            onClick: setRecID
-          }
-
-          return (
-            <div {...rowProps} key={`row-key-${rowIndex}`}>
```

Comment:
Spreading on ...rowProps was causing the issue of React not being able to find the className in time for the component to render. I'm not sure why this is happening but this is a workaround for now.